scripts/exportAssets.py

```diff
@@ -30,6 +30,16 @@ def check_elasticsearch_health():
         logger.warning('[exportAssets.py] Caught HTTP exception: {0}'.format(err))
         return False
 
+def asset_object_received(response_json):
+    if response_json['hits']['total']['value'] == 1:
```

Comment:
If the response doesn't come back with the right structure, this function will throw an exception. Are there circumstances where this response might not contain `['hits']['total']['value']`?

If you use the `dict` built-in `get()`, it will return `None` in the case of a missing key rather than throw an exception. Could do something like:
```
if response_json.get('hits'):
    if response_json['hits']['total']['value'] == 1:
            # do stuff
```
If 'hits' is in the response, then there better be total and value or ES is very broken...